documents/src/pages/elements/tree.md

```diff
@@ -286,7 +286,7 @@ if (tree) {
 
 ## Data property interface
 
-The `data ` property that passes through the `ef-tree` uses the [TreeData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts) interface for its data items.
+The `data ` property that passes through the `ef-tree` uses the [TreeData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts) interface for its data items. Each item defaults to type [TreeDataItem](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts), it can be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
also here

Reply:
Updated by following P'Ole suggestions of comboBox pattern.